Sendingway/Handler.cs

```diff
@@ -0,0 +1,87 @@
+using Dalamud.Game.Gui.PartyFinder.Types;
+using Dalamud.Plugin.Services;
+using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Net.Http;
+using System.Text;
+
+namespace Sendingway
+{
+    internal class Handler : IDisposable
+    {
+        private const string ServerUrl = "http://127.0.0.1:5000/";  // update this URL when we have the server
```

Comment:
we could potentially move this to its own constant file. also we should mark this with a big `// TODO: update this URL...` to make it stand out